Engine/Module/ModuleResourceManager.h

```diff
@@ -83,6 +96,7 @@ class ModuleResourceManager : public Module
 
 	std::unique_ptr<Importer> default_importer = std::make_unique<Importer>();
 	std::unique_ptr<ModelImporter> model_importer = nullptr;
+	std::unique_ptr<PrefabImporter> prefab_importer = nullptr;
```

Comment:
Maybe we should refactor all the pointers in the engine to have this structure? @mariofv

Reply:
Yes, atm the pointers are mixed between std::pointers and regular C++ pointers. We need to unify them into std::pointers at some point in the future to avoid memory leaks and other problems that could arrise easily.